presto-main/src/main/java/io/prestosql/sql/planner/LogicalPlanner.java

```diff
@@ -409,6 +414,80 @@ private RelationPlan createInsertPlan(Analysis analysis, Insert insertStatement)
                 statisticsMetadata);
     }
 
+    private RelationPlan createRefreshMVPlan(Analysis analysis, RefreshMV refreshMVStatement)
```

Comment:
There's a major revamp of the planner on the way, so you may want to rebase on that change to avoid conflicts later on: https://github.com/prestosql/presto/pull/3600

Reply:
ok, will do. Temporarily turning the PR to 'Draft' to make frequent changes and ensure that tests pass. Thanks.